C# Star Practices/SimpleCalculator/SimpleCalculator.Tests/Calculator.Operators.Test.cs

```diff
@@ -0,0 +1,68 @@
+namespace SimpleCalculator.Tests;
+
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.OperatorBusiness;
+using SimpleCalculator.Business.OperatorBusiness.Operators;
+
+// test base operators
+
+public class OperatorSumTest
```

Comment:
Has a lot of duplicate code with Calculator.Base.Test
you should only keep one of them